examples/basics/src/layouts/Layout.astro

```diff
@@ -22,12 +22,15 @@ const { title } = Astro.props;
 </html>
 <style is:global>
 	:root {
-		--accent: 124, 58, 237;
-		--accent-gradient: linear-gradient(45deg, rgb(var(--accent)), #da62c4 30%, white 60%);
+		--accent: 136, 58, 234;
+		--accent-light: 224, 204, 250;
+		--accent-dark: 49, 10, 101;
+		--accent-gradient: linear-gradient(45deg, rgb(var(--accent)), rgb(var(--accent-light)) 30%, white 60%);
 	}
 	html {
 		font-family: system-ui, sans-serif;
-		background-color: #F6F6F6;
+		background: #13151A url('astro_a.svg') center -32px no-repeat;
+		background-size: 224px;
```

Comment:
Hmm not sure if I love this! Maybe [inlining the SVG](https://yoksel.github.io/url-encoder/) to avoid having another file users have to delete?

Reply:
updated!